contracts/JBETHPaymentTerminalStore.sol

```diff
@@ -474,6 +474,11 @@ contract JBETHPaymentTerminalStore {
       ? _amount
       : PRBMathUD60x18.div(_amount, prices.priceFor(_currency, JBCurrencies.ETH));
 
+    // The amount being withdrawn must be at least as much as was expected.
+    if (_minReturnedWei > withdrawnAmount || balanceOf[_projectId] < withdrawnAmount) {
+      revert INADEQUATE_WITHDRAW_AMOUNT();
```

Comment:
hmm I think this error message only applies to the `_minReturnedWei > withdrawnAmount`  case

I think `INADEQUATE_PAYMENT_TERMINAL_STORE_BALANCE()` is more appropriate for the new case you've added

Reply:
Is it the withdrawAmount or the terminal balance fault :D ?

Reply:
maybe two separate checks?

Reply:
done!

Reply:
ah just saw this, thanks for handling it